.gitignore

```diff
@@ -296,3 +296,8 @@ certs
 
 # certificates and private keys in pem format
 *.pem
+
+# vk files
+*.vk
+*.sp1vk
+*.hex
```

Comment:
Thank god I was always having to manually remove them from my `git add .` every time I was doing local docker stuff.